docs/aro/setup-quay/quay-cli.md

```diff
@@ -0,0 +1,117 @@
+# Setting up Quay on an ARO cluster using Azure Container Storage 
+
+**Kristopher White x Connor Wooley**
+
+*07/25/2022*
+
+## Pre Requisites
+
+* an ARO cluster
+* oc/azure cli
```

Comment:
separate lines, they're two different CLIs,  link to the download or install instructions for each.